crates/sled/src/tree.rs

```diff
@@ -72,16 +72,38 @@ impl Tree {
     /// assert_eq!(t.set(&[1,2,3], vec![0]), Ok(None));
     /// assert_eq!(t.set(&[1,2,3], vec![1]), Ok(Some(IVec::from(&[0]))));
     /// ```
+    #[deprecated(since = "0.24.2", note = "replaced by `Tree::insert`")]
```

Comment:
I chose `0.24.2` but it is up to you to tell me the version where the deprecation begins.